site/src/i18n/en/common.json

```diff
@@ -1,17 +1,4 @@
 {
-  "workspaceStatus": {
```

Comment:
Aren't we still using these here? 

https://github.com/coder/coder/blob/main/site/src/components/WorkspaceStatusBadge/WorkspaceStatusBadge.tsx

Reply:
Oops, I thought they were coming entirely from the backend...

Reply:
Fixed!